x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/execute_action.tsx

```diff
@@ -30,14 +31,19 @@ export const ExecuteActionResult = memo<
     ResponseActionsExecuteParameters
   >
 >(({ command, setStore, store, status, setStatus, ResultComponent }) => {
+  const isSentinelOneV1Enabled = useIsExperimentalFeatureEnabled(
```

Comment:
Hmmm. Not sure using this feature flag here is appropriate. It currently applies only to Isolate and Release and we will likely have a different one when we get around to `execute` and other actions (same comment for other response actions below)

Reply:
Yeah that makes sense. I'll remove the feature flag check.

Reply:
updated b1b344e01783636a803ac4aa0cc6e54642af34eb